docs/api-docs/source/includes/_bitcoin_adapter.md

```diff
@@ -0,0 +1,646 @@
+# Bitcoind-adapter
+
+## List Addresses
+
+```shell
+curl -X POST
+  -d '{
+      "minConfirmations": 0,
+      "includeEmpty": true,
+      "includeWatchOnly": false
+    }'
+  -H "Content-Type: application/json"
+  "http://localhost:3009/v0/addresses/listaddresses"
+```
+
+```javascript
+const addressesService = require('@coinmesh/bitcoind-adapter').addressesService;
+
+addressesService.listReceivedByAddress(0, true, false).then(result => {
+  console.log(result);
+});
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{ "result":
+  [
+    {
+      "address": "mfh71i1bbXSoWsPeQ1Sqpy3ivy2zmtfng3",
```

Comment:
By chance were you able to update the addresses to valid bitcoin addresses?  Just curious.

Reply:
No.  Sorry.  I don't have an example of what that would look like.

Reply:
```
We also need to copy a line in here - https://github.com/coinmesh/coinmesh/blob/master/docs/api-docs/source/index.html.md
```

Could you do this one?  Not sure where it would go.

Reply:
Cool one thing that I was doing is running the node and then simply clicking on the wallet app to generate new address.  For the messages and stuff this was a bit more complex though.

Reply:
Derp.  That makes sense.  Should've just done that.